### DISCUSSION TITLE
Please add "Map absolute raw input to the osu! window" option back to lazer

### DISCUSSION THREAD

**Discussion by Buszek4** [ppy/osu#26231] [Ideas]
it's really hard to get the exact same tablet area as on stable without that opion, it would help older players a lot that got used to playing like this.
There's even was a reddit post about the issue. https://www.reddit.com/r/osugame/comments/18ma73j/cant_get_tablet_to_feel_right_in_lazer/

**Comment by peppy** [comment]
To clarify, you want to DISABLE this setting? Because the default way in lazer already acts as if it was enabled.

**Reply by SoarCompute** [reply]
Yes.
